cpp/unreal_plugins/SpEngine/Source/SpEngine/GameWorldService.h

```diff
@@ -41,6 +64,120 @@ class GameWorldService {
             SP_ASSERT(world_);
             return Unreal::toStdString(world_->GetName());
         });
+
+        unreal_entry_point_binder->bindFuncUnreal("game_world_service", "find_actors", [this]() -> std::vector<uint64_t> {
+            return Std::reinterpretAsVectorOf<uint64_t>(Unreal::findActors(world_));
+        });
+
+        unreal_entry_point_binder->bindFuncUnreal("game_world_service", "find_actors_as_map", [this]() -> TMapStringToInt {
+            return getIntMapFrom(Unreal::findActorsAsMap(world_));
+        });
+
+        unreal_entry_point_binder->bindFuncUnreal("game_world_service", "get_components", [this](const uint64_t& ptr) -> std::vector<uint64_t> {
```

Comment:
Don't call these function arguments `ptr`. Call them the same as what they're called in `Unreal.h`, e.g., `actor` or `parent` or something.